src/Facades/Forge.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Laravel\Forge\Facades;
+
+use Illuminate\Support\Facades\Facade;
+use Laravel\Forge\ForgeManager;
+
+/**
+ * @mixin \Laravel\Forge\Forge
```

Comment:
I can add a GitHub action to generate all the methods in the doc block here if you'd prefer (using the official [Facade Documenter](https://github.com/laravel/facade-documenter)), or just push them in this PR. Lemme know! 👍

Reply:
@stevebauman yeah, let's do that, please!